syndesis/tests/org.fusesource.ide.syndesis.extension.core.tests/src/test/java/org/fusesource/ide/syndesis/extensions/core/model/SyndesisExtensionSqlConnectorJSONLoadingTest.java

```diff
@@ -0,0 +1,172 @@
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import org.fusesource.ide.syndesis.extensions.core.model.ActionDataShape;
+import org.fusesource.ide.syndesis.extensions.core.model.PropertyDefinitionStep;
+import org.fusesource.ide.syndesis.extensions.core.model.SyndesisAction;
+import org.fusesource.ide.syndesis.extensions.core.model.SyndesisActionDescriptor;
+import org.fusesource.ide.syndesis.extensions.core.model.SyndesisExtension;
+import org.fusesource.ide.syndesis.extensions.core.model.SyndesisExtensionProperty;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author lhein
+ */
+public class SyndesisExtensionSqlConnectorJSONLoadingTest {
+
+	private SyndesisExtension extension;
+
+	@Before
+	public void onStart() throws IOException {
+		File jsonFile = new File("testdata/sql-connector.json");
+		this.extension = SyndesisExtension.getJSONFactoryInstance(new FileInputStream(jsonFile));
+	}
+
+	/**
+	 * Test method for {@link org.fusesource.ide.syndesis.extensions.core.model.SyndesisExtension#getId()}.
+	 */
+	@Test
+	public void testGetId() {
+		assertEquals("Id value is not matching", "sql", extension.getId());
+	}
+
+	/**
+	 * Test method for {@link org.fusesource.ide.syndesis.extensions.core.model.SyndesisExtension#getIcon()}.
+	 */
+	@Test
+	public void testGetIcon() {
+		assertEquals("icon value is not matching", "fa-database", extension.getIcon());
+	}
+
+	/**
+	 * Test method for {@link org.fusesource.ide.syndesis.extensions.core.model.SyndesisExtension#getDescription()}.
+	 */
+	@Test
+	public void testGetDescription() {
+		assertEquals("description value is not matching", "Invoke SQL or stored procedures.", extension.getDescription());
+	}
+	
+	/**
+	 * Test method for {@link org.fusesource.ide.syndesis.extensions.core.model.SyndesisExtension#getConnectorCustomizers()}.
+	 */
+	@Test
+	public void testGetConnectorCustomizers() {
+		assertThat(extension.getConnectorCustomizers().contains("io.syndesis.connector.sql.customizer.DataSourceCustomizer"));
+	}
+	
+	/**
+	 * Test method for {@link org.fusesource.ide.syndesis.extensions.core.model.SyndesisExtension#getDependencies()}.
+	 */
+	@Test
+	public void testGetDependencies() {
+		assertThat(extension.getDependencies()).filteredOn("id", "io.syndesis.connector:connector-sql:1.3-SNAPSHOT").isNotNull(); 
+	}
+
+	/**
+	 * Test method for {@link org.fusesource.ide.syndesis.extensions.core.model.SyndesisExtension#getActions()}.
+	 */
+	@Test
+	public void testGetActions() {
+		assertTrue("actions are not correct", extension.getActions() != null && !extension.getActions().isEmpty() && extension.getActions().size() == 4);
+	}
+	
+	@Test
+	public void testGetName() {
+		assertThat(extension.getName()).isEqualTo("Database");
+	}
+	
+	@Test
+	public void testGetTags() {
+		assertThat(extension.getTags().contains("verifier"));
```

Comment:
the parenthesis is not at the right place

`assertThat(extension.getTags()).contains("verifier");`